### DISCUSSION TITLE
How does this compare against using ClamAV directly?

### DISCUSSION THREAD

**Discussion by physkets** [kpcyrd/libredefender#45] [Q&A]
In what ways does libredefender make the scanning experience better, when compared to using the clamav service, timer and manual command-line scans?

This isn't quite clear to me from the README.

**Comment by tongpu** [comment]
From a technical point of view there's not a huge difference. One benefit that I see is that I can always ask libredefender about the status of the last scan, list problems detected, but for me the main difference is that I get a desktop notification if any threats are detected (which was one false positive in the 2 years I've been using it :smile:).

**Comment by wpeckham** [comment]
The advantage is less to the user than to companies with a security policy. This ensures that the AV scan runs on time even if the user forgets or never figured out it should be run in the first place. (Yes, we have those, and some are managers. The rest are mostly sales staff. Don't ask.)
